cmd/kaf/produce.go

```diff
@@ -46,6 +47,8 @@ func init() {
 	produceCmd.Flags().StringVar(&timestampFlag, "timestamp", "", "Select timestamp for record")
 	produceCmd.Flags().Int32VarP(&partitionFlag, "partition", "p", -1, "Partition to produce to")
 
+	produceCmd.Flags().IntVarP(&schemaID, "schema-id", "", -1, "Value schema id for avro messsage encoding")
```

Comment:
what about changing this to --avro-schema-id ? i think this is more in line with other flags like --proto-*